src/MessageContainer.js

```diff
@@ -173,27 +168,35 @@ export default class MessageContainer extends React.Component {
 
 const styles = StyleSheet.create({
   container: {
-    flex: 1
+    flex: 1,
   },
   notInvertedContentContainerStyle: {
-    justifyContent: 'flex-end'
-  }
+    justifyContent: 'flex-end',
+  },
 });
 
 MessageContainer.defaultProps = {
   messages: [],
   user: {},
   renderFooter: null,
   renderMessage: null,
-  onLoadEarlier: () => {
-  },
+  renderLoadEarlier: null,
+  onLoadEarlier: () => {},
+  inverted: true,
```

Comment:
This seems like a potential behavior change, why was a new default prop added?

Reply:
Fixed in Commit 4571ddabf60e5dad54c2d83970c7c5e15097c88c